### PR TITLE
Create works and About me sections

### PR DESCRIPTION
**Pull request by emmiiorji**
I implemented the 'works' and 'About me' sections of the portfolio's mobile version following the [Figma design template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)

Find some screenshots below:

image
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
       <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
     </ul>
   </section>
-  
+
+  <section class="works-section">
+    <h1>My Recent Works</h1>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that your page could have only one `h1` header. All other headers should be `hx` (h2, h3, etc) - please, consider it for the whole page 👍

File: index.html [hunk 2]
```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
...
+      <li>HTML</li>
+      <li>CSS</li>
+    </ul>
+
+    <div class="title-container"><div class="square-icon"></div><h2>Frameworks</h2></div>
+    <ul>
+      <li>Bootstrap</li>
+      <li>Ruby on Rails</li>
+      <li>RSPec</li>
+      <li>CapyBara</li>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that the "About me" section has to be fixed by alignment - please, refer to the Figma design
> Your version
Screen Shot 2022-03-26 at 20 46 31

> Figma version
Screen Shot 2022-03-26 at 20 43 03

- [ ] Please, consider the "About me" section paragraph content is a little bit hard to read - try to fix the position of the paragraph, to not cover the background
Screen Shot 2022-03-26 at 20 50 57

File: index.html [hunk 3]
```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
...
+  <section class="about-me">
+    <h1>About me</h1>
+    <p>
+      Hello I’m a software developer! I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.
+    </p>
+    <button type="button">Get My Resume</button>
+    <hr>
```
Comment (elmar8287) [inline index.html:164]: - [ ] Please, consider the button styling for the "Get My Resume" button: Active, Hover, and Pressed

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Informative PR description 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @emmiiorji, 

Your project is complete! Nice job fixing the comments from the previous code reviewer :fireworks:  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional but important
- [ ] It would be better if we include a detailed instruction about how to clone and run the app locally :100: I highly recommend you to fix that

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (emmiiorji) [reply index.html]: Hi @elmar8287 

Thanks for a detailed and kind review.

Concerning having a single h1 header on a page, can you kindly take a look at the screenshot below from [this article](http://diveintohtml5.it/semantics.html) and let me know what you think?

image

Thank you.

**Pull request merged**